github/messages.go

```diff
@@ -175,10 +177,15 @@ func ValidatePayload(r *http.Request, secretKey []byte) (payload []byte, err err
 		return nil, fmt.Errorf("Webhook request has unsupported Content-Type %q", ct)
 	}
 
-	sig := r.Header.Get(signatureHeader)
-	if err := ValidateSignature(sig, body, secretKey); err != nil {
-		return nil, err
+	// Only validate the signature if a secret message exists. This is intended for
+	// local development only and all webhooks should contain a secret message.
+	if secretKey != nil {
```

Comment:
In Go, it is more idiomatic with slices to check against a length of zero which will work for both nil and a zero-length slice. Checking against `nil` is discouraged because it doesn't work for a zero-length slice.

So please change this to `if len(secretKey) > 0 {`.